script/Arcueid/Day 4/04_00_ARC04_9_1.txt

```diff
@@ -26,7 +26,7 @@ I look around the room for anything I could use as a weapon, but find nothing.
 [sha:587799b98da2e39bffe63d3f3abe823bed7dc0f0]{
 -- Page 724, Offset 7506. Mods: @k, @e.
 -- 「……様子を見てくる。
-"...I'm going to find out what's happening.
+"I'm going to go scope things out.
```

Comment:
this feels a little awkward. The old one is a little more unwieldy, but it feels more natural.